tests/federation_room_join_test.go

```diff
@@ -459,6 +459,9 @@ func TestSendJoinPartialStateResponse(t *testing.T) {
 
 	srv := federation.NewServer(t, deployment,
 		federation.HandleKeyRequests(),
+
+		// accept incoming presence transactions, etc
+		federation.HandleTransactionRequests(nil, nil),
```

Comment:
some explanation of where the presence is coming from would be nice!

Reply:
that would require me to figure that out. Tbh I just cargo-culted all the other tests in this file...